pages/03.Tutorials/18.create-backup-and-restore/default.en.md

```diff
@@ -25,38 +26,88 @@ $ kubectl create namespace backup-tutorial
 namespace/backup-tutorial created
 ```
 
-For our tutorial we will just deploy a NGINX Hello World app, run:
+For our tutorial we will deploy an NGINX container with a persistent volume named nginx-logs. It is important to note the annotation which is added to the pod spec. Without the annotation your volume will not be included in the backup task. The annotation must include the names of the volumes to backup.
+
+Note: hostPath volumes are not supported, but the new local volume type is supported.
```

Comment:
This could use links to the Kubernetes documentation to pages that explain what hostPath and local volumes are